hoot-core/src/main/cpp/hoot/core/io/OsmXmlReader.cpp

```diff
@@ -448,27 +449,34 @@ bool OsmXmlReader::startElement(const QString & /* namespaceURI */,
     else if (qName == QLatin1String("nd") && _element)
     {
       long ref = _parseLong(attributes.value("ref"));
```

Comment:
Added this section to allow OsmXmlReader to load element child refs, even when they're not present in the output to support ExternalMergeElementSorter.  Obviously, if the child refs never appear in the actual final sorted output, that's a problem.  Subsequent reading from the sorted stream served up by ExternalMergeElementSorter would log warnings that revealed the problem.